internal/code-generator/generator/linguist-commit.go

```diff
@@ -0,0 +1,17 @@
+package generator
+
+import (
+	"bytes"
+	"text/template"
+)
+
+// Commit takes in commit and builds source file from tmplPath. It complies with type File signature.
```

Comment:
This still sounds a bit weird to me _Commit takes **a** commit and builds **the** source file from tmplPath_ sounds better.